.github/workflows/rules.yml

```diff
@@ -32,45 +31,45 @@ jobs:
       run:
         shell: bash
     steps:
-    - uses: actions/checkout@v3
-    - name: Set up Python ${{ inputs.python-version }}
-      uses: actions/setup-python@v4
-      with:
-        python-version: ${{ inputs.python-version }}
-    - name: Install package
-      run: |
-        python -m pip install --upgrade pip
-        pip install pytest-cov
-        pip install pytest-env
-        pip install ".[${{ inputs.pip-extras }}]"
-        if [[ "${{ inputs.environment }}" == "qibo" ]]; then
-          pip install git+https://github.com/qiboteam/qibojit
-          pip install tensorflow
-        fi
-        if [[ "${{ inputs.environment }}" == "qibolab" || \
-              "${{ inputs.environment }}" == "qibojit" ]]; then
-          pip install git+https://github.com/qiboteam/qibo
-        fi
-        pip install pylint
-    - name: Test with pylint
-      run: |
-        pylint src --errors-only
-        pylint src --exit-zero
-    - name: Test with pytest core
-      run: |
-        pytest
-    - name: Test example
-      if: ${{ inputs.doctests }}
-      run: |
-        sudo apt install pandoc
-        make -C doc doctest
-        OMP_NUM_THREADS=1 pytest examples/
-    - name: Upload coverage to Codecov
-      if: startsWith(runner.os, 'Linux')
-      uses: codecov/codecov-action@v3
-      with:
-        token: ${{ secrets.CODECOV_TOKEN }}
-        file: ./coverage.xml
-        flags: unittests
-        name: unit-tests
-        fail_ci_if_error: true
+      - uses: actions/checkout@v3
```

Comment:
Do you really need those extra spaces?

Reply:
I believe the answer to be "no", you don't need.

But I don't fight against formatters, this is just what [Prettier](https://prettier.io/) likes (and I won't start a crusade to save a couple of spaces - in case you use a different formatter, I can also switch).